src/common/components/guidance-links.tsx

```diff
@@ -1,14 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { NewTabLink } from 'common/components/new-tab-link';
+import { LinkComponentType } from 'common/types/link-component-type';
+import { ElectronExternalLink } from 'electron/views/device-connect-view/components/electron-external-link';
 import { isEmpty } from 'lodash';
 import * as React from 'react';
 import { HyperlinkDefinition } from 'views/content/content-page';
+
 import { NamedFC } from '../react/named-fc';
-import { NewTabLink } from './new-tab-link';
 
 export interface GuidanceLinksProps {
     links: HyperlinkDefinition[];
     classNameForDiv?: string;
+    LinkComponent?: LinkComponentType;
```

Comment:
I think making this optional makes it easier to write the same class of bug (forgetting to use the electron-specific link in the electron app) easier to do in the future. I think it looks reasonably contained to make it non-optional and pipe it in everywhere; it looks like there are only 3 other places where `<GuidanceLinks>` is being used, one of which already has a `LinkComponent` dep in place and the other 2 of which already have `deps` chains in place.